content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -209,6 +217,10 @@ are enabled. For details on that, see the `--feature-gates` and `--runtime-confi
 [kube-apiserver parameters](/docs/reference/command-line-tools-reference/kube-apiserver/).
 kube-scheduler, kube-controller-manager and kubelet also need the feature gate.
 
+When a resource driver reports the status of the devices, then the
+`DRAResourceClaimDeviceStatus` feature gate has to be enabled in addition to
+`DynamicResourceAllocation`.
+
```

Comment:
Same here: a sub-section dedicated specifically to this feature might work better.

Reply:
Done, thank you for the comment and sorry for the delay.